inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/sources/TestMqttReader.java

```diff
@@ -106,13 +107,13 @@ public void setUp() throws Exception {
 
         when(jobProfile.get(eq(CommonConstants.PROXY_INLONG_GROUP_ID), anyString())).thenReturn(groupId);
```

Comment:
Sorry, it is recommended to put the reader's logic in the source. The current reader needs to be deleted after it

Reply:
sorry, do you mean write the MqttReader's code in the mqttsource file? But how to handle the split function, just rerturn null as other sources?

Reply:
> sorry, do you mean write the MqttReader's code in the mqttsource file? But how to handle the split function, just rerturn null as other sources?

yes

Reply:
ok, i will fix it